LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animation-important-001-expected.txt

```diff
@@ -1,9 +1,9 @@
 
 PASS Interpolated value is observable
-FAIL Important rules override animations assert_equals: expected "rgb(0, 128, 0)" but got "rgb(15, 15, 15)"
-FAIL Non-overriden interpolations are observable assert_equals: expected "rgb(0, 128, 0)" but got "rgb(15, 15, 15)"
+PASS Important rules override animations
+PASS Non-overriden interpolations are observable
 FAIL Important rules override animations (::before) assert_equals: expected "rgb(0, 128, 0)" but got "rgb(15, 15, 15)"
```

Comment:
Do you have an idea why this doesn't pass now. Is `!important` not a cascade issue?

Reply:
There is a FIXME about the pseudo elements.